r/fratPhyloseq.Rmd

```diff
@@ -0,0 +1,63 @@
+---
+title: "Phyloseqify the Fratjuice samples"
+author: "Richard Sprague"
+date: "1/5/2018"
+output: github_document
+---
+
+Load the prerequisites:
+
+```{r}
+library(phyloseq)
+library(actino)
+library(tidyverse)
+SAMPLEDIR <- "../samples/ubiome/ubiome-processed-outputs"
+
+```
+
+Read the mapping data into a more familiar "Actino-style" mapfile
+
+```{r}
+results <- readxl::read_excel(paste0(SAMPLEDIR,"/results.xlsx"))
+ssrs <- results$SeqID
+username <- results$Sample_ID
+mapping <- data.frame(SSR=ssrs,id=username)
+```
+
+Save it as an Excel file to be later readable by the actino functions.
+You only have to run this chunk once. It's commented now because I already created the map file.
+
+```{r}
+#library(xlsx)
+#xlsx::write.xlsx(mapping, file = paste0(SAMPLEDIR,"/json/fratmap.xlsx"), row.names = FALSE)
+```
+
+Now create the Phyloseq object
+```{r, message= FALSE}
+sampleFiles<-actino::just_json_files_in(paste0(SAMPLEDIR,"/json"))
+fratMap <- paste0(SAMPLEDIR,"/json/fratmap.xlsx")
```

Comment:
Let's stick with a variable name convention of lowecase with underscores for spaces... e.g. `frat_map` and `fratjuice_phylum`.

I also think it's helpful for readers to note the object type when possible... so I'd do `mapping_df` rather than `mapping`.